src/Loader.php

```diff
@@ -246,7 +246,19 @@ protected function sanitiseVariableValue($name, $value)
                 /mx',
                 $quote
             );
+            $originalValue = $value;
             $value = preg_replace($regexPattern, '$1', $value);
+            if(($e = preg_last_error()) !== PREG_NO_ERROR) {
+                $l = get_defined_constants(true);
+                $l = $l['pcre'];
+                $l = array_flip($l);
+                throw new InvalidFileException($l[$e]);
```

Comment:
I don't think the original error is going to make sense. Can we prepend some other message on the front?

Reply:
I think we could, I'll think of something.